src/app/issuer/components/issuer-create/issuer-create.component.scss

```diff
@@ -5,6 +5,23 @@
 	height: 20rem
 }
 
+::ng-deep .forminput .forminput-x-label {
```

Comment:
This does seem to improve the font, however now the padding between the headline and the image field seems to be less than for the other fields.